Find a functionally equivalent code:android.graphics.Picture.createFromStream:COMMENT
Method Modifier: public      static      deprecated  
Comment:/**
 * Create a new picture (already recorded) from the data in the stream. This
 * data was generated by a previous call to writeToStream(). Pictures that
 * have been persisted across device restarts are not guaranteed to decode
 * properly and are highly discouraged.
 *
 * <p>
 * <strong>Note:</strong> a picture created from an input stream cannot be
 * replayed on a hardware accelerated canvas.
 *
 * @see #writeToStream(java.io.OutputStream)
 * @deprecated The recommended alternative is to not use writeToStream and
 * instead draw the picture into a Bitmap from which you can persist it as
 * raw or compressed pixels.
 */

@@ -1,3 +1,3 @@
 {
-    return new Picture(nativeCreateFromStream(stream, new byte[WORKING_STREAM_STORAGE]), true);
+    return new Picture(nativeCreateFromStream(stream, new byte[WORKING_STREAM_STORAGE]));
 }

